src/qwen3_tts_test/voice_clone_batch.py

```
@@ -0,0 +1,133 @@
+
+    if not ref_audio.exists():
+        logger.error("参照音声が見つかりません: %s", ref_audio)
+        raise FileNotFoundError(f"ref audio not found: {ref_audio}")
+    if not text_file.exists():
+        logger.error("テキストファイルが見つかりません: %s", text_file)
+        raise FileNotFoundError(f"text file not found: {text_file}")
+    if not out_path.parent.exists():
+        try:
+            out_path.parent.mkdir(parents=True, exist_ok=True)
+            logger.info("出力ディレクトリ作成: %s", out_path.parent)
+        except Exception as exc:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)